src/apps/front-office/home/pages/HomePage/sections/DealsDayTwo/TopRate.tsx

```diff
@@ -15,19 +15,19 @@ export default function TopRate(attributes: TopRateProps) {
       <div>
         <TopSellingCard
           productImageTop={productImg10}
-          productDescription="Nestle Original Coffee-Mate Coffee Creamer"
+          productName="Nestle Original Coffee-Mate Coffee Creamer"
           salePrice={32.85}
           priceOld={33.8}
         />
         <TopSellingCard
           productImageTop={productImg11}
-          productDescription="Nestle Original Coffee-Mate Coffee Creamer"
```

Comment:
We don't pass properties, we pass a `product object` prop directly

Reply:
this section is still in process. Also, this section on the `homepage`  is waiting for when finished.